test/main.go

```diff
@@ -364,3 +365,27 @@ func csharpTests(ctx context.Context, root *dagger.Container, t *testCase) error
 
 	return err
 }
+
+
+// kotlinAndroidTests run the java integration tests suite against a container running Flipt.
+func kotlinAndroidTests(ctx context.Context, root *dagger.Container, t *testCase) error {
+	path := "x86_64"
+
+	if architecture == "arm64" {
+		path = "arm64-v8a"
+	}
+
+	_, err := root.From("gradle:8.5.0-jdk11").
+		WithWorkdir("/src").
+		WithDirectory("/src", t.dir.Directory("flipt-client-kotlin-android"), dagger.ContainerWithDirectoryOpts{
+			Exclude: []string{"./.idea/", ".gradle/", "build/"},
+		}).
+		WithFile(fmt.Sprintf("/src/main/cpp/libs/%s/libfliptengine.so", path), t.test.File(libFile)).
```

Comment:
is `src/main/cpp/libs` the standard path for android / kotlin to look for shared C libs? or is this configured somewhere else?

Reply:
sirce I'm using .a file , it is configured in the CMakeLists.txt. there's no standard for .a file, unlike .so file that must be stored in jniLibs directory

Reply:
i see are you saying this is just for the tests? just wondering why its at `src/main/cpp/libs/%s/libfliptengine.so` if its a `.a` file

Reply:
well, I thought I could use it just for test. but, I've realized that it cannot be done. it should be .a instead. even for testing